recipes/jupyter-resource-usage/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "jupyter-resource-usage" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ade39a081bbc9104bf7d6a54d68e90b021d7e53c1b0e218d1457ee7a8f1134a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - jupyter-packaging >=0.7.9
+  run:
+    - notebook
+    - psutil
+    - python >=3.6
+
+test:
+  imports:
+    - jupyter_resource_usage
```

Add `pip check` to check for consistency to the pip requirements.
```suggestion
    - jupyter_resource_usage
  requires:
    - pip
  commands:
    - pip check
```

---

Thanks!